deployment-jvm/local-jvm/performance/createPerformanceData.sh

```diff
@@ -16,7 +16,7 @@
 FILE=performance/target/performance-*-exec.jar
```

Comment:
Minor note, I've seen the use of JARFILE which seems like a better description of what it is.

Reply:
updated